chef/cookbooks/crowbar/templates/default/crowbar-delete-pacemaker-resources.sh.erb

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# Old pacemaker resources must be deleted in the old setup before they get created from upgraded node
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+UPGRADEDIR=/var/lib/crowbar/upgrade
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+echo "Executing $BASH_SOURCE"
+
+set -x
+
+mkdir -p $UPGRADEDIR
+
+if [[ -f $UPGRADEDIR/pacemaker-resources-delete-ok ]] ; then
+    echo "Pacemaker resources already successfully deleted"
+    exit 0
+fi
+
+rm -f $UPGRADEDIR/pacemaker-resources-delete-failed
+
+<% if @use_ha %>
+
+echo "Stopping pacemaker resources..."
+
+# Delete order constraints before stopping the resources
+# so they do not cause starting of something that was already stopped.
+
+for order in $(crm configure show | grep ^order | cut -d " " -f2); do
+    crm configure delete $order
+done
```

Comment:
Not  even this seems to help. After one execution of the script, keystone service is still running....